main/TMP1075.c

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+#include "esp_log.h"
+#include "driver/i2c.h"
+
+#include "TMP1075.h"
+
+#define I2C_MASTER_SCL_IO 48        /*!< GPIO number used for I2C master clock */
```

Comment:
all these defines should only be needed in the main/i2c_master.c, no more in the dirvers